test/opexecuter.cpp

```diff
@@ -320,8 +321,17 @@ void OpExecuter::ExecuteOnNGraph(vector<Tensor>& ngraph_outputs) {
   }
 
   // Create nGraph backend
-  // Create the nGraph backend
-  auto backend = ng::runtime::Backend::create("CPU");
+  // If NGRAPH_TF_BACKEND is set create that backend
+  // Else create backend of type ng_backend_name
+  const char* ng_backend_env_value = std::getenv("NGRAPH_TF_BACKEND");
+  if (ng_backend_env_value != nullptr) {
+    string backend_env = std::string(ng_backend_env_value);
+    if (!backend_env.empty()) {
+      ng_backend_name = backend_env;
+    }
```

Comment:
before setting a backend, we could check if that string is supported using ngraph_backends_len 
 + ngraph_list_backends
```
https://github.com/NervanaSystems/ngraph-tf/blob/master/src/ngraph_api.cc#L18
```

Reply:
NGraph throws an error if we pass an invalid string. Had the same concern, so checked this scenario.
The error looks like this with NGRAPH_TF_BACKEND=U
```
unknown file: Failure
C++ exception with description "Backend 'U' not registered. Error:/localdisk/shrestha/tf-1.11/ngraph-tf/build/ngraph/ngraph_dist/lib/libu_backend.so: cannot open shared object file: No such file or directory" thrown in the test body.
```